src/wagtail_live/models.py

```diff
@@ -5,7 +5,8 @@
 from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
 from wagtail.core.fields import StreamField
 
-from .blocks import LivePostBlock
+from wagtail_live.blocks import LivePostBlock
+from wagtail_live.signals import live_page_update
```

Comment:
Do you mean that I should add the handler of the signal in the ready method and only define the signal itself in `signals.py`?

Reply:
Yes I think you should consider that. Because this way it might be imported more than once, which causes the function to be connected multiple times possibly.

Reply:
I've followed your suggestion. I tried to add test for it but it fails with django 2.2. Do you have an idea why this happens?